content/docs/components-and-props.md

```diff
@@ -154,11 +154,11 @@ function Comment(props) {
 
 [](codepen://components-and-props/extracting-components)
 
-It accepts `author` (an object), `text` (a string), and `date` (a date) as props, and describes a comment on a social media website.
+यह स्वीकार करता है `author` (एक ऑब्जेक्ट), `text` (एक string), और `date` (एक date) जैसे props, और सोशल मीडिया वेबसाइट पर एक टिप्पणी का वर्णन करता है|
 
-This component can be tricky to change because of all the nesting, and it is also hard to reuse individual parts of it. Let's extract a few components from it.
+यह रेंडरस को बदलने के लिए मुश्किल हो सकता है क्योंकि एक के अंदर दूसरी संरचना के वजह से,तथा इसकी अकेली पार्ट्स का पुन: उपयोग करना भी कठिन है| चलो कुछ कौम्पोनॅन्टस निकालते हैं|
```

Comment:
`रेंडरस` => `कौम्पोनॅन्टस`
at many places components is translated to this, please update it everywhere

Reply:
You have translated `components` to `रेंडरस` at many places, I meant to change at those places